sortfilterproxymodel.qbs

```diff
@@ -1,56 +1,62 @@
-import qbs 1.0;
+import qbs 1.0
 
 Product
 {
-    name: "sortfilterproxymodel";
+    name: "sortfilterproxymodel"
     type: "dynamiclibrary"
-    targetName: "sortfilterproxymodel";
-    destinationDirectory: project.deployDirectory;
+    targetName: "sortfilterproxymodel"
 
-    readonly property stringList qtModules : ["core", "qml", "gui"];
+    destinationDirectory: project.deployDirectory
+
+    readonly property stringList qtModules:
+    [
+        "core",
+        "qml",
+        "gui"
+    ]
 
     cpp.defines:
     {
-        var defines = [];
+        var defines = []
         
-        defines.push('QT_NO_KEYWORDS');
-        defines.push('QRMVISUALCONFIG_LIBRARY_BUILD');
+        defines.push('QT_NO_KEYWORDS')
+        defines.push('SORT_FILTER_PROXY_MODEL_LIBRARY_BUILD')
 
         if (qbs.buildVariant === "release")
         {
-            defines.push('QT_NO_DEBUG_OUTPUT=1');
+            defines.push('QT_NO_DEBUG_OUTPUT=1')
         }
         
-        return defines;
+        return defines
     }
 
-    cpp.includePaths: ["."];
+    cpp.includePaths: [ "." ]
```

Comment:
Last week I looked at a bunch of code formatters. astyle and clang-format I think match a lot of our requirements. At least in letting us specify these spaces/indentation issues.

For QML I found none though. Maybe once we are done with C++ code formatting.. we can look into the QML one, by using/editing an existing JS formatter (there are tons of these btw.)

~1% of the commits in avalon currently are simple style fixes btw.
````
$ git log --pretty=oneline | wc -l
2400
$git log --pretty=oneline | grep -i style | wc -l
20
````